_script/cibuild-docker

```diff
@@ -38,11 +38,20 @@ function arguments_mgmt() {
 }
 
 function travis_deploy_flags() {
-    exec_env_vars=""
+    docker_deploy_flags=""
     if $(test "$sim_deploy" = "yes")
     then
-        echo "INFO: deploy flag enabled. We will enable the env variables to simulate a travis merge to master"
-        exec_env_vars='-e TRAVIS_PULL_REQUEST="false" -e TRAVIS_BRANCH="master"'
+        echo "INFO: deploy flag enabled. We will activate the env variables, and mount your docker config/credentials,\
+         to simulate a travis merge to master"
+        local_docker_config_store="${HOME}/.docker"
+        if ! $(test -f "${local_docker_config_store}/config.json")
+        then
+            echo "ERROR: You need a valid docker config credentials file in your ${local_docker_config_store} directory\
+             in order to deploy new images, with read access to the user $travis (uid 2000 in the container). \
+             You can generate it using docker login"
+            exit 1
+        fi
+        docker_deploy_flags="-v ${local_docker_config_store}:${travis_home}/.docker -e TRAVIS_PULL_REQUEST=false -e TRAVIS_BRANCH=master"
```

Comment:
why do you need the TRAVIS_PULL_REQUEST and TRAVIS_BRANCH in the "docker run"  command?

Reply:
In case I want to simulate a deploy (so the job travis will run when a PR is integrated against master), these are the env variables that advise this is the scenario